### PR TITLE
[release-1.8] cleanup extensions config

### PR DESCRIPTION
**Pull request by kyessenov**
Manual cherry-pick instead of https://github.com/istio/istio/pull/28314

### CODE REVIEW

**Review by bianpengyuan — APPROVED** [review]

**Comment by bianpengyuan** [comment]
The test failure is real: https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio/28409/integ-multicluster-k8s-tests_istio_release-1.8/1321893805644845056/artifacts/multicluster-base-6de8a9af234b4/TestMulticlusterReachability/reachability/echo-2-0-%3ehttp:/echo-2-0:http/_test_context/cluster-local-2-38329-state192422608/cluster-0/local-v1-545db69bfb-njb4v_istio-proxy.log

File: manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml [hunk 1]
```diff
@@ -386,13 +382,12 @@ spec:
...
                     {
-                      "debug": "false",
-                      "stat_prefix": "istio"{{- if .Values.global.multiCluster.clusterName }},
+                      {{- if .Values.global.multiCluster.clusterName }}
                       "metrics": [
                         {
                           "dimensions": {
-                            "destination_cluster": "node.metadata['CLUSTER_ID']",
                             "source_cluster": "downstream_peer.cluster_id"
+                            "destination_cluster": "node.metadata['CLUSTER_ID']",
```
Comment (bianpengyuan) [inline manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml:390]: Invalid json here.

Comment (kyessenov) [reply manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml]: Thanks, fixed.

**Pull request merged**